kuma/static/js/analytics.js

```diff
@@ -107,16 +118,16 @@
 
         trackLink: function(event, url, data) {
             // ctrl or cmd click or context menu
-            var newTab = (event.metaKey || event.ctrlKey || event.type == 'contextmenu');
+            var newTab =
```

Comment:
It looks weird to have this on a line by itself. Can we solve the line length problem a different way?